.github/workflows/circleci.yml

```diff
@@ -1,6 +1,10 @@
 on: [status]
+permissions:
+  contents: read
 jobs:
   circleci_artifacts_redirector_job:
+    permissions:
+      statuses: write
```

Comment:
Does this actually need `write` permissions?

Reply:
I think yes - see https://github.com/larsoner/circleci-artifacts-redirector-action/blob/master/index.js#L75 and https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-statuses

Reply:
Yeah, seems to me well after looking at your links. Thanks for looking it up.